examples/jsm/loaders/LDrawLoader.js

```diff
@@ -1949,7 +1969,9 @@ class LDrawLoader extends Loader {
 			.parseModel( text, this.materialLibrary )
 			.then( group => {
 
+				this.applyMaterialsToMesh( group, MAIN_COLOUR_CODE, this.materialLibrary, true );
 				this.computeConstructionSteps( group );
+				group.userData.fileName = "";
```

Comment:
Here should also be single quotes.

Reply:
Thanks, I'll commit when the clone issue is decided.